libraries/rib-rx2/src/main/java/com/badoo/ribs/rx2/workflows/RxWorkflowActivity.kt

```diff
@@ -0,0 +1,34 @@
+package com.badoo.ribs.rx2.workflows
+
+import android.content.Intent
+import android.os.Bundle
+import com.badoo.ribs.android.RibActivity
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+
+abstract class RxWorkflowActivity : RibActivity() {
```

Comment:
Do you think we still need this class?
I can't find any usages of `workflowFactory` in our code base.
Personally I think it does not gives clients any particular value and reason to use it. It is quite simple to check `intent` in any way you want from client code and handle just it. Also this class has some flaws like: only  `ACTION_VIEW` is supported, no `onNewIntent` handling, RxJava support only, requirement to handle the node instance manually after creation (`workflowFactory` does not provide the node).

Reply:
Yeah that's a fair point. Missing support can be added though. So far all of this still existed, but as part of RibActivity. Now at least it's extracted as a concern. I'd still rather keep it to provide a starting point.